pretyping/recordops.mli

```diff
@@ -23,7 +23,7 @@ type proj_kind = {
   pk_canonical: bool;
 }
 
-val mk_proj_kind : Name.t -> bool -> proj_kind
+val mk_proj_kind : Name.t -> bool -> bool -> proj_kind
```

Comment:
Maybe label the bool arguments so we can differentiate them.
Or remove the `mk_proj_kind` so people have to use the record building syntax `{pk_name = ...; ...}`.